content/docs/reference-react-dom.md

```diff
@@ -12,12 +12,18 @@ If you load React from a `<script>` tag, these top-level APIs are available on t
 
 The `react-dom` package provides DOM-specific methods that can be used at the top level of your app and as an escape hatch to get outside of the React model if you need to. Most of your components should not need to use this module.
 
-- [`render()`](#render)
-- [`hydrate()`](#hydrate)
-- [`unmountComponentAtNode()`](#unmountcomponentatnode)
-- [`findDOMNode()`](#finddomnode)
+- [`createRoot()`](#createroot)
+- [`hydrateRoot()`](#hydrateroot)
 - [`createPortal()`](#createportal)
 
+> Note
+> 
+> These methods are considered legacy. Both `render` and `hydrate` will warn that your app is running in React 17:
```

Comment:
I think what this means is that React is only rendering synchronously? "running in React 17" is not that clear to me.

Reply:
They both render synchronously (createRoot by itself doesn't opt into concurrency), so I think this is primarily about disabling automatic batching. I agree "in React 17" is confusing. We should make sure that, whichever way we word it, your upgrade to 18 is not considered "completed" until you've switched away from render.

Reply:
@gaearon are you saying that we should include upgrade instructions here? My goal in including this line was to give  heads up about the warning you'll see in the console which is:

> ReactDOM.hydrate is no longer supported in React 18. Use hydrateRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more: https://reactjs.org/link/switch-to-createroot

Updated to clarify a bit better, let me know if that works now.

Reply:
Sweet!